vendor/modules.txt

```diff
@@ -10,6 +10,8 @@ github.com/PuerkitoBio/urlesc
 github.com/asaskevich/govalidator
 # github.com/beorn7/perks v1.0.1
 github.com/beorn7/perks/quantile
+# github.com/caarlos0/env v3.5.0+incompatible
+github.com/caarlos0/env
```

Comment:
Where is it used?

Reply:
Its used here: https://github.com/percona/percona-xtradb-cluster-operator/blob/master/cmd/pitr/main.go#L12